src/ships.js

```diff
@@ -1,698 +1,75 @@
-		return "Raider";
-	}
-	static get short_name() {
-		return "R";
-	}
-	static get cost() {
-		return 12;
-	}
-	static get hullSize() {
-		return 2;
-	}
-	static get requiredShipSizeTechnology() {
-		return 1;
-	}
-	static get requiredMinesTechnology() {
-		return 0;
-	}
-	static get requiredMineSweeperTechnology() {
-		return 0;
-	}
-	static get requiredFightersTechnology() {
-		return 0;
-	}
-	static get requiredCloakingTechnology() {
-		return 1;
-	}
-	static get maxQuantity() {
-		return 50;
-	}
-}
-
-
-export class Carrier {
-	static get type() {
-		return "Carrier";
-	}
-	static get name() {
-		return "Carrier";
-	}
-	static get short_name() {
-		return "CV";
-	}
-	static get cost() {
-		return 12;
-	}
-	static get hullSize() {
-		return 1;
-	}
-	static get requiredShipSizeTechnology() {
-		return 1;
-	}
-	static get requiredMinesTechnology() {
-		return 0;
-	}
-	static get requiredMineSweeperTechnology() {
-		return 0;
-	}
-	static get requiredFightersTechnology() {
-		return 1;
-	}
-	static get requiredCloakingTechnology() {
-		return 0;
-	}
-	static get maxQuantity() {
-		return 50;
-	}
-}
-
-
-export class FighterOne {
-	static get type() {
-		return "FighterOne";
-	}
-	static get name() {
-		return "Fighter 1";
-	}
-	static get short_name() {
-		return "F";
-	}
-	static get cost() {
-		return 5;
-	}
-	static get hullSize() {
-		return 1;
-	}
-	static get requiredShipSizeTechnology() {
-		return 1;
-	}
-	static get requiredMinesTechnology() {
-		return 0;
-	}
-	static get requiredMineSweeperTechnology() {
-		return 0;
-	}
-	static get requiredFightersTechnology() {
-		return 1;
-	}
-	static get requiredCloakingTechnology() {
-		return 0;
-	}
-	static get maxQuantity() {
-		return 50;
-	}
-}
-
-
-export class FighterTwo {
-	static get type() {
-		return "FighterTwo";
-	}
-	static get name() {
-		return "Fighter 2";
-	}
-	static get short_name() {
-		return "F";
-	}
-	static get cost() {
-		return 5;
-	}
-	static get hullSize() {
-		return 1;
-	}
-	static get requiredShipSizeTechnology() {
-		return 1;
-	}
-	static get requiredMinesTechnology() {
-		return 0;
-	}
-	static get requiredMineSweeperTechnology() {
-		return 0;
-	}
-	static get requiredFightersTechnology() {
-		return 2;
-	}
-	static get requiredCloakingTechnology() {
-		return 0;
-	}
-	static get maxQuantity() {
-		return 50;
-	}
-}
-
-
-export class FighterThree {
-	static get type() {
-		return "FighterThree";
-	}
-	static get name() {
-		return "Fighter 3";
-	}
-	static get short_name() {
-		return "F";
-	}
-	static get cost() {
-		return 5;
-	}
-	static get hullSize() {
-		return 1;
-	}
-	static get requiredShipSizeTechnology() {
-		return 1;
-	}
-	static get requiredMinesTechnology() {
-		return 0;
-	}
-	static get requiredMineSweeperTechnology() {
-		return 0;
-	}
-	static get requiredFightersTechnology() {
-		return 3;
-	}
-	static get requiredCloakingTechnology() {
-		return 0;
-	}
-	static get maxQuantity() {
-		return 50;
-	}
-}
+export class Ship {
```

Comment:
Check the indentation for that file.

Reply:
I'll fix that, I thought I had converted tabs to two spaces, but apparently in this file that didn't happen.